generator/resources/ec2_stress_test_matrix.json

```diff
@@ -37,7 +34,6 @@
         "username": "ec2-user",
         "installAgentCommand": "go run ./install/install_agent.go rpm",
         "ami": "cloudwatch-agent-integration-test-al2*",
-        "caCertPath": "/etc/ssl/certs/ca-bundle.crt",
```

Comment:
I still see `caCertPath` in `type matrixRow struct`. Could it be removed? Should it?

Reply:
I see some other tests still has `caCertPath` in the .json so I don't think we should remove it.
e.g.:
- https://github.com/aws/amazon-cloudwatch-agent-test/blob/e07fe7adb1b1d75244d8984507d3f83a7237c3d3/generator/resources/ec2_linux_test_matrix.json#L8
- https://github.com/aws/amazon-cloudwatch-agent-test/blob/e07fe7adb1b1d75244d8984507d3f83a7237c3d3/generator/resources/ec2_gpu_test_matrix.json#L8
-

Reply:
It been used for local stack to generate CA Bundle for the [CA Bundle test when there is region](https://github.com/aws/amazon-cloudwatch-agent-test/tree/main/test/ca_bundle) which can not use the auto CA bundle. However, for your answer, we cannot remove it.